docs/semgrep-ci/manual-ci-setup.md

```diff
@@ -0,0 +1,507 @@
+---
+slug: manual-ci-setup
+append_help_link: true
+title: Running Semgrep in CI without Semgrep App 
+description: ""
+hide_title: true
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Running Semgrep in continuous integration (CI) without Semgrep App
+
+Run Semgrep in your Continuous Integration (CI) pipeline to scan your repository for code vulnerabilities and other issues. This guide explains how to set up Semgrep in your pipeline without the use of Semgrep App, also known as a **stand-alone** setup.
+
+There are three general steps to setting up Semgrep in your CI pipeline:
+
+1. Set up the CI job or action to scan with Semgrep and receive an exit code.
+2. Test that the Semgrep job is scanning your repository and that you are able to view scan results in your CI provider's log.
+3. Refine the CI job's parameters. It is easier to troubleshoot any parameters after testing that the job runs successfully.
+
+![Steps to run Semgrep in CI without Semgrep App](/img/semgrep-ci-overview-noapp.png "Steps to integrate Semgrep in CI without Semgrep App")
+*Figure 1. Steps to run Semgrep in CI manually.*
+
+This guide defines a job or CI job as a script executed within a certain environment and managed by a CI provider.
+
+By refining a job's parameters, you are able to achieve the following goals:
+
+* **Run Semgrep on a schedule.** Run full scans on mainline branches at the least intrusive time on developer teams.
+* **Run Semgrep with custom rules.** Apply rules specific to your organization's business goals and coding conventions.
+* **Run Semgrep when an event triggers.** Run Semgrep when a pull or merge request (PR or MR) is created. These event triggers or event hooks are dependent on your CI provider. 
+* **Run Semgrep on relevant files and blocks of code.** Configure Semgrep to ignore files and folders such as test files, configuration files, and files from other vendors.
+* **Configure a Semgrep CI job to pass even when any finding is detected.** By default, manual configurations pass when any finding is detected. You can also configure Semgrep to fail CI jobs when findings are reported.
+* **Output, export, or save findings to a file.** Semgrep can save to a number of file formats, including SARIF and JSON. 
+
+## Limitations of manually configured Semgrep CI scans 
+
+* Findings are not tiered based on a user-defined criteria, such as the severity of a finding, and actions cannot be undertaken based on this criteria. Alternatively, Semgrep App's [Rule Board](/semgrep-app/rule-board) can tier rules and undertake actions, such as blocking PRs or MRs on a finding generated by a reliable rule.
```

Comment:
"Tiers of findings are not based..." is incorrect. The whole point is that findings are not tiered because they mechanism doesn't exist. That means that yes, any finding is blocking. There is no tier in non-app-connected scans.

Action I can undertake is to I'll make the sentences in active.

Reply:
Thank you for clarifying this. The active voice certainly helps. Maybe (just an idea) again specifying that this is not an issue with Semgrep App, but only in stand-alone setup makes sense, but this is optional and I will leave this up to you. @s-santillan

Reply:
But we specify this above the list, so feel free to ignore me here. @s-santillan